lib/reform/form/validation/unique_validator.rb

```diff
@@ -33,7 +38,12 @@ def validate_each(form, attribute, value)
     model = form.model_for_property(attribute)
 
     # search for models with attribute equals to form field value
-    query = model.class.where(attribute => value)
+    if options[:case_sensitive] == false
+      downcased_value = value.nil? ? value : value.downcase
```

Comment:
True enough, unless you are running Ruby 2.4.0. Is there a solution for
earlier versions?

2016-07-09 22:40 GMT+01:00 Abdelkader Boudih <notifications@github.com>:

> In lib/reform/form/validation/unique_validator.rb
> <https://github.com/trailblazer/reform-rails/pull/10#discussion_r70172368>
> :
>
> > @@ -33,7 +38,12 @@ def validate_each(form, attribute, value)
> >      model = form.model_for_property(attribute)
> >
> >      # search for models with attribute equals to form field value
> > -    query = model.class.where(attribute => value)
> > +    if options[:case_sensitive] == false
> > +      downcased_value = value.nil? ? value : value.downcase
>
> hmm. This will not support all unicode characters.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/trailblazer/reform-rails/pull/10/files/bd4ebd23dd00dcf82efc9744bf1777246ed37d31#r70172368>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJfZU8G1oXqTepvrohA9_DgILBzFKhyDks5qUBU8gaJpZM4JItDe>
> .
>